pkg/proxy/iptables/proxier.go

```diff
@@ -1179,7 +1164,7 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 
 		// Capture nodeports.
-		if svcInfo.NodePort() != 0 && len(nodeAddresses) != 0 {
```

Comment:
I think that this was needed because of the port holder

Reply:
ah, maybe